phing/tasks/blt.xml

```diff
@@ -35,6 +35,8 @@
 
   <target name="blt:rsync-template" hidden="true">
     <echo>Copying files from BLT's template into your project...</echo>
+    <!-- Remove the README.md before rsync -->
```

Comment:
This will be run every time that somebody performs an update on their local project. Instead, we should put this in the create-project target.